e2e/pages/events/createOrderEvent.page.js

```diff
@@ -136,6 +142,12 @@ module.exports = {
     });
   },
 
+  async enterDateOfIssue(date) {
+    I.fillField(this.fields.dateOfIssue.day, date.day);
```

Comment:
Seeing as we are starting to enter date a lot, do you think it wise we add a new method to the steps.js (and other parts where needed, not sure) that populates a date/datetime field given a date?

Reply:
This might be a good idea, and we could make it handle dates and date+time. I'll take a look